README.md

```diff
@@ -3,4 +3,38 @@
 A curated list of amazingly awesome resources for (not only) LMC developers.
 
 ## Table of Contents
-To Be Done
+- [Development](#development)
+- [Core review](#code-review)
+- [Q&A](#qa)
+- [Git](#git)
+- [Books](#books)
+- [Newsletter](#newsletters)
+
+## Development
+- [@Ocramius - Fluent interfaces are evil](https://ocramius.github.io/blog/fluent-interfaces-are-evil/)
+- Matthias Noback - article series
+    - https://www.ibuildings.nl/blog/2016/01/programming-guidelines-php-developers-part-1-reducing-complexity
+    - https://www.ibuildings.nl/blog/2016/01/programming-guidelines-php-developers-part-2-getting-rid-null
+    - https://www.ibuildings.nl/blog/2016/08/why-write-tests
+- [Symfony komponenty](https://www.zdrojak.cz/serialy/symfony-po-kruckach/)
```

Comment:
I'd create somethinglike "PHP specific" subsection as part of this section and put this Symfony topic here (And I think more will come). The other links are language-independent.